index.html

```diff
@@ -54,23 +54,23 @@ <h2 class="second-header">I'm a Software Developer</h2>
         <ul class="my-icons-list">
           <li class="icon-item1 iconMedia">
             <a href="https://github.com/Alejandroq12"><img src="./assets/images/main_section/github.png"
-                alt="github" /></a>
+                alt="Julio GitHub account" /></a>
```

Comment:
## Image text alternatives: 
All images have an alt attribute which provides alternative text for screen readers. However, some of the alt text could be improved to be more descriptive. For example, the alt text for the social media icons could be more specific than just the name of the platform.
-- Changes suggested.
-- We fixed it.